Uncaught Error: Cannot find module 'module'

I see checkbox near `module` in [deno_std/node](https://github.com/denoland/deno_std/tree/main/node#supported-builtins) and file [module.ts](https://github.com/denoland/deno_std/blob/main/node/module.ts) exist.

When I run such code:

```js
import { createRequire } from "https://deno.land/std@0.88.0/node/module.ts";

const require = createRequire(import.meta.url);
require('module');
```

Using:

```sh
 deno run --unstable --allow-read mod.js
```

Everything is crashed with

```js
Require stack:
- /Users/coderaiser/putout/packages/putout/mod.js
      const err = new Error(message) as Error & {
                  ^
    at Function._resolveFilename (https://deno.land/std@0.88.0/node/module.ts:270:19)
    at Function._load (https://deno.land/std@0.88.0/node/module.ts:377:29)
    at Module.require (https://deno.land/std@0.88.0/node/module.ts:130:21)
    at require (https://deno.land/std@0.88.0/node/module.ts:1141:16)
    at file:///Users/coderaiser/putout/packages/putout/mod.js:5:1
    at <anonymous> (<anonymous>)
```

I'm wandering is it a bug or `module` not supported yet?

It used for example in such popular library as [v8-compile-cache](https://www.npmjs.com/package/v8-compile-cache).

```js
error: Uncaught Error: Cannot find module 'module'
Require stack:
- /Users/coderaiser/putout/node_modules/v8-compile-cache/v8-compile-cache.js
- /Users/coderaiser/putout/packages/putout/lib/cli/index.js
- /Users/coderaiser/putout/packages/putout/mod.js
      const err = new Error(message) as Error & {
                  ^
    at Function._resolveFilename (https://deno.land/std@0.88.0/node/module.ts:270:19)
    at Function._load (https://deno.land/std@0.88.0/node/module.ts:377:29)
    at Module.require (https://deno.land/std@0.88.0/node/module.ts:130:21)
    at require (https://deno.land/std@0.88.0/node/module.ts:1141:16)
    at Object.<anonymous> (file:///Users/coderaiser/putout/node_modules/v8-compile-cache/v8-compile-cache.js:3:16)
    at Module._compile (https://deno.land/std@0.88.0/node/module.ts:165:36)
    at Object.Module._extensions..js (https://deno.land/std@0.88.0/node/module.ts:1092:10)
    at Module.load (https://deno.land/std@0.88.0/node/module.ts:144:34)
    at Function._load (https://deno.land/std@0.88.0/node/module.ts:410:14)
    at Module.require (https://deno.land/std@0.88.0/node/module.ts:130:21)
```

--- a/node/module.ts
+++ b/node/module.ts
@@ -613,6 +613,7 @@ nativeModulePolyfill.set(
   createNativeModule("events", nodeEvents),
 );
 nativeModulePolyfill.set("fs", createNativeModule("fs", nodeFS));
+nativeModulePolyfill.set("module", createNativeModule("module", Module));
 nativeModulePolyfill.set("os", createNativeModule("os", nodeOs));
 nativeModulePolyfill.set("path", createNativeModule("path", nodePath));
 nativeModulePolyfill.set(

--- a/node/module_test.ts
+++ b/node/module_test.ts
@@ -91,6 +91,7 @@ Deno.test("requireNodeJsNativeModules", () => {
   require("crypto");
   require("events");
   require("fs");
+  require("module");
   require("os");
   require("path");
   require("querystring");
@@ -130,6 +131,7 @@ Deno.test("native modules are extensible", () => {
     "crypto",
     "events",
     "fs",
+    "module",
     "os",
     "path",
     "querystring",

